modules/telco-core-software-stack.adoc

```diff
@@ -14,10 +14,13 @@ The Red{nbsp}Hat telco core {product-version} solution has been validated using
 |Component |Software version
 
 |{rh-rhacm-first}
-|2.12^1^
+|2.13
+
+|{gitops-title}
+|1.16
 
 |Cluster Logging Operator
-|6.1^2^
+|6.2
```

Comment:
Updated because I can see the Operator is released and compatible with 4.18: https://access.redhat.com/support/policy/updates/openshift_operators?extIdCarryOver=true&sc_cid=7013a000003StsiAAC#platform-agnostic